File: skellig-test-step-reader/src/main/java/org/skellig/teststep/reader/model/TestStepFileExtension.java
package org.skellig.teststep.reader.model;

public enum TestStepFileExtension {
    STS("sts");

    private String extensionName;

    TestStepFileExtension(String extensionName) {
        this.extensionName = extensionName;
    }

    public String getName() {
        return extensionName;
    }
}

File: skellig-test-step-runner/src/main/kotlin/org/skellig/teststep/runner/model/TestStepFileExtension.kt
package org.skellig.teststep.runner.model;

public enum TestStepFileExtension {
    STS("sts");

    private String extensionName;

    TestStepFileExtension(String extensionName) {
        this.extensionName = extensionName;
    }

    public String getName() {
        return extensionName;
    }
}
